JetLagged/build.gradle.kts

```diff
@@ -14,13 +14,40 @@
  * limitations under the License.
  */
 
+import com.diffplug.gradle.spotless.SpotlessExtension
+
 plugins {
     alias(libs.plugins.gradle.versions)
     alias(libs.plugins.version.catalog.update)
     alias(libs.plugins.android.application) apply false
     alias(libs.plugins.kotlin.android) apply false
     alias(libs.plugins.kotlin.parcelize) apply false
     alias(libs.plugins.compose) apply false
+    alias(libs.plugins.spotless)
```

Comment:
One of the issues with Spotless is that it's not compatible with the Gradle configuration cache. This is why it should only be applied using an init script, rather than being part of the project build config.